servers/rendering/renderer_viewport.cpp

```diff
@@ -753,6 +757,8 @@ void RendererViewport::draw_viewports(bool p_swap_buffers) {
 	//draw viewports
 	RENDER_TIMESTAMP("> Render Viewports");
 
+	GodotProfileZoneGrouped(_profile_zone, "render viewports");
+
 	//determine what is visible
 	draw_viewports_pass++;
 
```

Comment:
Again, can't put this one to the right line, but we should have a separate slice for each rendered viewport:

```cpp
	for (int i = 0; i < sorted_active_viewports.size(); i++) {
		GodotProfileZone("render viewport");
		Viewport *vp = sorted_active_viewports[i];
```